packages/manager/apps/web/client/app/domain/general-informations/domain-general-informations.controller.js

```diff
@@ -93,8 +97,21 @@ export default class DomainTabGeneralInformationsCtrl {
       refreshAlert: false,
     };
     this.vm = {
-      protection: { uiSwitch: {} },
-      dnssec: { uiSwitch: {} },
+      protection: {
+        disabled: [
+          PROTECTION_TYPES.LOCKING,
+          PROTECTION_TYPES.UNLOCKING,
+          PROTECTION_TYPES.UNAVAILABLE,
+        ].includes(this.domain.protection),
+        uiSwitch: {},
```

Comment:
Not yours but isn't `uiSwitch` just a boolean ? Moreover could it be something less related to UI implemation ? Something like `enabled` ?

Reply:
Yes you're right, I'll change the name